src/Files.App/Utils/Archives/ArchiveHelpers.cs

```diff
@@ -108,6 +109,7 @@ public static async Task CompressArchiveAsync(IArchiveCreator creator)
 					FileOperationType.Compressed
 				);
 			}
+			await associated.Refresh_Click();
```

Comment:
Wouldn't it be better to only update the individual item? Ideally this would be done from the file watcher but I don't think we need to resort to refreshing the tab.

Reply:
@Mohammed-619

Reply:
@yaira2 I would like to give a shot at fixing this issue, and continue from where Mohammed brought it. Could you point me to where the file watcher code is in this repo?

Reply:
@TheChilledBuffalo thank you for reaching out, the file watcher code is in ItemViewModel.cs.